.github/workflows/codeql.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
+    timeout-minutes: 15
```

Comment:
Can't comment there but up in the file you're using `7.x.x` instead of `2.x.x`, i assume due to a copy-pasta error from a project like Soto?

Reply:
Removed